### PR TITLE
Fix code imports

### LINKED ISSUE #1384: Fix broken imports in SDK and check docs
**Describe the bug**

Some core imports are broken in the current SDK docs.

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Started fixing code imports as the line numbers are outdated.

## Links to any relevant issues

Fixes #1384.

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [ ] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: docs/build/iota-sdk/1.0/docs/how-tos/alias/create.mdx [hunk 1]
```diff
@@ -55,22 +55,23 @@ The following example will:
 
 1. Instantiate a [`Wallet`](https://docs.rs/iota-sdk/latest/iota_sdk/wallet/core/struct.Wallet.html), get Alice's `Account` which was
    [created in the first guide](../accounts-and-addresses/create-account.mdx) and [sync it](../accounts-and-addresses/check-balance.mdx).
-2. Create an Alias Output transaction by calling the [`Account.prepareCreateAliasOutput()`](../../references/nodejs/classes/Account.md#preparecreatealiasoutput) function, and then [send](../../references/nodejs/classes/Account.md#send) it.
+2. Create an Alias Output transaction by calling the [`Account.createAliasOutput()` TODO Fix](../../references/nodejs/classes/Account.md#createaliasoutput) function.
```
Comment (lucas-tortora) [inline docs/build/iota-sdk/1.0/docs/how-tos/alias/create.mdx:58]:
```suggestion
2. Create an Alias Output transaction by calling the [`Account.createAliasOutput()` ](../../references/nodejs/classes/Account.md#createaliasoutput) function. 
```

File: docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/melt.mdx [hunk 2]
```diff
@@ -46,14 +46,13 @@ https://github.com/iotaledger/iota-sdk/blob/develop/sdk/examples/how_tos/native_
 
 1. Instantiate a [`Wallet`](https://docs.rs/iota-sdk/latest/iota_sdk/wallet/core/struct.Wallet.html), get Alice's `Account` which was
    [created in the first guide](../accounts-and-addresses/create-account.mdx) and [sync it](../accounts-and-addresses/check-balance.mdx).
-2. Prepare to melt `MELT_AMOUNT` native tokens transaction by calling the [`Account.prepareMeltNativeToken()`](../../references/nodejs/classes/Account.md#preparemeltnativetoken)
-   function anb send the prepared transaction using the [`Account.send()`](../../references/nodejs/classes/Account.md#send)
+2. Create a transaction to melt `MELT_AMOUNT` native tokens by calling the [`Account.meltNativeToken()`](../../references/nodejs/classes/Account.md#meltnativetoken) <!-- TODO Update ref link -->
```
Comment (lucas-tortora) [inline docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/melt.mdx:49]:
```suggestion
2. Create a transaction to melt `MELT_AMOUNT` native tokens by calling the [`Account.meltNativeToken()`](../../references/nodejs/classes/Account.md#meltnativetoken) 
```

File: docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/send.mdx [hunk 3]
```diff
@@ -82,19 +82,18 @@ https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how
...
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/native_tokens/send.ts#L38-L43
+https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/native_tokens/send.ts#L43-L48
 ```
 
 </div>
 
-4. Prepare the transaction that will send the tokens using the [`Account.prepareSendNativeTokens()`](../../references/nodejs/classes/Account.md#preparesendnativetokens)
-   function and send the prepared transaction using the [`Account.send()`](../../references/nodejs/classes/Account.md#send)
-   function.
+4. Create the transaction that will send the tokens using the [`Account.sendNativeTokens()` TODO](../../references/nodejs/classes/Account.md#account.sendnativetokens)
```
Comment (lucas-tortora) [inline docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/send.mdx:90]:
```suggestion
4. Create the transaction that will send the tokens using the [`Account.sendNativeTokens()`](../../references/nodejs/classes/Account.md#account.sendnativetokens)
```

File: docs/build/iota-sdk/1.0/docs/how-tos/nfts/mint-nft.mdx [hunk 4]
```diff
@@ -108,19 +108,18 @@ https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how
...
 ```typescript reference
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/nfts/mint_nft.ts#L55-L62
+https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/nfts/mint_nft.ts#L61-L74
 ```
 
 </div>
 
-4. Prepare the transaction that will mint the NFTs using the [`Account.prepareMintNfts()`](../../references/nodejs/classes/Account.md#preparemintnfts)
-   function and send the prepared transaction using the [`Account.send()`](../../references/nodejs/classes/Account.md#send)
+4. Create the transaction that will mint the NFTs using the [`Account.mintNfts()`](../../references/nodejs/classes/Account.md#mintnfts) <!-- TODO Update ref link -->
```
Comment (lucas-tortora) [inline docs/build/iota-sdk/1.0/docs/how-tos/nfts/mint-nft.mdx:116]:
```suggestion
4. Create the transaction that will mint the NFTs using the [`Account.mintNfts()`](../../references/nodejs/classes/Account.md#mintnfts) 
```

File: docs/build/iota-sdk/1.0/docs/how-tos/nfts/send-nft.mdx [hunk 5]
```diff
@@ -77,19 +77,17 @@ https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how
...
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/nfts/send_nft.ts#L45-L50
+https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/nfts/send_nft.ts#L53-L58
 ```
 
 </div>
 
-4. Prepare the transaction that will send the NFT using the [`Account.prepareSendNft()`](../../references/nodejs/classes/Account.md#preparesendnft) function
-   and send the transaction using the [`Account.send()`](../../references/nodejs/classes/Account.md#send)
-   function.
+4. Create the transaction that will send the NFT using the [`Account.sendNft()` TODO](../../references/nodejs/classes/Account.md#sendnft) function.
```
Comment (lucas-tortora) [inline docs/build/iota-sdk/1.0/docs/how-tos/nfts/send-nft.mdx:85]:
```suggestion
4. Create the transaction that will send the NFT using the [`Account.sendNft()`](../../references/nodejs/classes/Account.md#sendnft) function.
```

File: docs/build/iota-sdk/1.0/docs/how-tos/accounts-and-addresses/consolidate-outputs.mdx [hunk 6]
```diff
@@ -69,40 +69,29 @@ https://github.com/iotaledger/iota-sdk/blob/develop/sdk/examples/how_tos/account
...
-```typescript reference
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/consolidate-outputs.ts#L70-L72
-```
-
-</div>
-
-4. Send the `Transaction` by calling the [`PreparedTransaction.send()`](../../references/nodejs/classes/PreparedTransaction.md#send)
-   function.
+3. Send a [`Transaction`](../../references/nodejs/classes/Transaction.md) to consolidate the Outputs by calling
+   the [`Account.consolidateOutputs()`](../../references/nodejs/classes/Account.md#consolidateoutputs) <!-- TODO Update ref link --> function.
```
Comment (lucas-tortora) [inline docs/build/iota-sdk/1.0/docs/how-tos/accounts-and-addresses/consolidate-outputs.mdx:78]:
```suggestion
   the [`Account.consolidateOutputs()`](../../references/nodejs/classes/Account.md#consolidateoutputs) function.
```

**Review by lucas-tortora — CHANGES_REQUESTED** [review]

File: docs/build/iota-sdk/1.0/docs/how-tos/alias/create.mdx (hunk 1 above)
Comment (Dr-Electron) [reply docs/build/iota-sdk/1.0/docs/how-tos/alias/create.mdx:58]:
```suggestion
2. Create an Alias Output transaction by calling the [`Account.createAliasOutput()` TODO Fix](../../references/nodejs/classes/Account.md#createaliasoutput) <!-- TODO Update ref link --> function.
```

File: docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/send.mdx (hunk 3 above)
Comment (Dr-Electron) [reply docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/send.mdx:90]:
```suggestion
4. Create the transaction that will send the tokens using the [`Account.sendNativeTokens()`](../../references/nodejs/classes/Account.md#account.sendnativetokens) <!-- TODO Update ref link -->
```

File: docs/build/iota-sdk/1.0/docs/how-tos/nfts/send-nft.mdx (hunk 5 above)
Comment (Dr-Electron) [reply docs/build/iota-sdk/1.0/docs/how-tos/nfts/send-nft.mdx:85]:
```suggestion
4. Create the transaction that will send the NFT using the [`Account.sendNft()`](../../references/nodejs/classes/Account.md#sendnft) <!-- TODO Update ref link --> function.
```

File: docs/build/iota-sdk/1.0/docs/how-tos/accounts-and-addresses/consolidate-outputs.mdx (hunk 6 above)
Comment (Dr-Electron) [reply docs/build/iota-sdk/1.0/docs/how-tos/accounts-and-addresses/consolidate-outputs.mdx:78]: We should leave them until I fix the refs

File: docs/build/iota-sdk/1.0/docs/how-tos/nfts/mint-nft.mdx (hunk 4 above)
Comment (Dr-Electron) [reply docs/build/iota-sdk/1.0/docs/how-tos/nfts/mint-nft.mdx:116]: We should leave them until I fix the refs

File: docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/melt.mdx (hunk 2 above)
Comment (Dr-Electron) [reply docs/build/iota-sdk/1.0/docs/how-tos/native-tokens/melt.mdx:49]: We should leave them until I fix the refs

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**